data/src/main/scala/ch.epfl.scala.index.data/download/PlayWsDownloader.scala

```diff
@@ -57,10 +34,9 @@ trait PlayWsDownloader {
    *   }
    * }}}
    */
-  def managed[A](f: WSClient => Future[A]): Future[A] =
-    Future(wsClient).flatMap(
-      client => f(client).andThen { case _ => client.close() }
-    )
+  def managed[A](f: WSClient => Future[A]): Future[A] = {
+    PlayWsClient.open().toFuture.flatMap(f)
```

Comment:
Where is the `toFuture` method defined? i’m looking at http://jsuereth.com/scala-arm/latest/api/index.html#resource.ManagedResource but that may not be the right place…?

Reply:
It should be there... but it's not, so I think the scaladoc is not up-to-date.
You can find the `toFuture` documentation [here](https://github.com/jsuereth/scala-arm/blob/7a157f96cf2a906d803c6a7110e68145a44c61ca/src/main/scala/resource/ManagedResource.scala#L112-L118).